cfme/tests/configure/test_multi_tenancy_manual.py

```diff
@@ -0,0 +1,434 @@
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [pytest.mark.ignore_stream("upstream")]
+
+
+@pytest.mark.manual
```

Comment:
Required:
This file is for `manual tests` so better to have this `global marker`

Reply:
Done, along with manual, i have marked @test_requirements.cfme_tenancy as well, Cause the file is dedicated for manual tests related to multi-tenancy only.